client/src/components/Tickets/TicketModal.js

```diff
@@ -33,7 +33,18 @@ import './TicketModal.scss';
 const formSchema = Yup.object().shape({
   reply: Yup.string().required('Required')
 });
-
+const Attachment = ({ attachment }) => (
+  <div>
+    <ul>
+      {attachment.slice(1).map(attachmentName => (
+        <li> {attachmentName} </li>
+      ))}
+    </ul>
+  </div>
+);
+Attachment.propTypes = {
+  attachment: PropTypes.string.isRequired
+};
```

Comment:
Can you add some newlines for visual separation?
```suggestion

const Attachment = ({ attachment }) => (
  <div>
    <ul>
      {attachment.slice(1).map(attachmentName => (
        <li> {attachmentName} </li>
      ))}
    </ul>
  </div>
);
Attachment.propTypes = {
  attachment: PropTypes.string.isRequired
};

```